src/Tribe/Commerce/PayPal/Custom_Argument.php

```diff
@@ -57,6 +77,18 @@ public static function decode( $encoded, $assoc_array = false ) {
 			$encoded = str_replace( '\"', '"', $encoded );
 		}
 
+		// in case we receive the param in non json 'user_id:0,tribe_handler:tpp,pid:4' format
+		if ( false === strpos( $encoded, '{' ) && false === strpos( $encoded, '}' ) ) {
```

Comment:
why not try to `json_decode` and check if it's false? The PHP implementation will make the same check you are doing at a much lower, and faster, level